src/Aspire.Dashboard/Components/Pages/StructuredLogs.razor

```diff
@@ -76,7 +76,7 @@
             else
             {
                 <div slot="end">
-                    <FluentCounterBadge HorizontalPosition="70" Count="@(ViewModel.Filters.Count)" Appearance="Appearance.Accent">
+                    <FluentCounterBadge HorizontalPosition="70" Count="@(ViewModel.Filters.Count(filter => filter.Enabled))" Appearance="Appearance.Accent">
```

Comment:
There are a lot of `filter => filter.Enabled` in this code. What about adding `EnabledFilters` property to the view model, e.g.
```cs
public IEnumerable<...> GetEnabledFilters() => Filters.Count(filter => filter.Enabled);
```

Reply:
We would need to add this to the like 5 view models that contain a filter list. How about this instead?

```csharp
public static IEnumerable<TelemetryFilter> GetEnabledFilters(this IEnumerable<TelemetryFilter> filters)
{
  return filters.Where(filter => filter.Enabled);
}
```

Reply:
Sure